.github/workflows/testing.yml

```diff
@@ -11,7 +11,7 @@ jobs:
 
     strategy:
       matrix:
-        os: [ubuntu-22.04, ubuntu-20.04]
+        os: [ubuntu-24.04]
```

Comment:
If possible, I would like you to use 22.04 and 20.04 in addition to 24.04 because they are a platforms where the current u-tags are tested.

Reply:
After rethinking, I found ctags side should use 24.04.
Please, forget this.

Reply:
Oh, you already added a change for "older distros". Thank you.

Reply:
I tried, but there are some problems with it. Clang dependencies are different on older ubuntus and pegof freezes during test, so there might be more bugs lurking in the code. 

I'll probably merge the basic part for Ubuntu 24.4 which is working fine and leave the rest until I can properly fix it in the older distros.